frappe/desk/query_report.py

```diff
@@ -195,6 +195,7 @@ def run(
 	parent_field=None,
 	are_default_filters=True,
 ):
+	validate_filters_permissions(report_name, filters, user)
```

Comment:
The `user` variable can be None here, which is bad: reports fail when there are filters, but work fine without filters :laughing:

Reply:
fixes in https://github.com/frappe/frappe/pull/28468